contributingGuides/STYLE.md

```diff
@@ -1000,16 +1007,6 @@ class Rey extends Jedi {
 }
 ```
 
-## ESNext: Are we allowed to use [insert new language feature]? Why or why not?
-
-JavaScript is always changing. We are excited whenever it does! However, we tend to take our time considering whether to adopt the latest and greatest language features. The main reason for this is **consistency**. We have a style guide so that we don't have to have endless conversations about how our code looks and can focus on how it runs.
-
-So, if a new language feature isn't something we have agreed to support it's off the table. Sticking to just one way to do things reduces cognitive load in reviews and also makes sure our knowledge of language features progresses at the same pace. If a new language feature will cause considerable effort for everyone to adapt to or we're just not quite sold on the value of it yet, we won't support it.
-
-Here are a couple of things we would ask that you *avoid* to help maintain consistency in our codebase:
-
-- **Async/Await** - Use the native `Promise` instead
```

Comment:
I agree with removing this part, but I'd still prefer keeping the rest since it is helpful context about adopting any new feature.